### PR TITLE
Zoomeye

### PR DESCRIPTION
**Pull request by e2002e**
Here is an unique branch for the zoomeye module update.

### CODE REVIEW

File: modules/auxiliary/gather/zoomeye_search.rb [hunk 1]
```diff
@@ -41,8 +40,7 @@ def initialize(info = {})
     )
     register_options(
       [
-        OptString.new('USERNAME', [true, 'The ZoomEye username']),
-        OptString.new('PASSWORD', [true, 'The ZoomEye password']),
+        OptString.new('ZOOMEYE_APIKEY', [true, 'The ZoomEye api key']),
```
Comment (bwatters-r7) [inline modules/auxiliary/gather/zoomeye_search.rb:43]: Why are we removing the login?  If we want to add the API Key as an access vector, that makes sense, but why remove existing functionality?

Comment (e2002e) [reply modules/auxiliary/gather/zoomeye_search.rb:43]: There was changes regarding the api access, among which login has been disabled in favor of a token authentication.

**Comment by e2002e** [comment]
I contacted the zoomeye support and they gave me a week api plan for testing.
It's working well.

**Comment by adfoster-r7** [comment]
@e2002e Is it possible to share the API key for testing? Or if you can provide the output of this module being run and working as expected to help get this landed

**Comment by e2002e** [comment]
> @e2002e Is it possible to share the API key for testing? Or if you can provide the output of this module being run and working as expected to help get this landed

Here: A8E19E9C-A0EF-E29fc-983a-ed2ffcaf5f0

**Comment by adfoster-r7** [comment]
## Release Notes

Updates the `modules/auxiliary/gather/zoomeye_search` module to work again

**Comment by adfoster-r7** [comment]
Thanks!

**Pull request merged**